libraries/standard/mqtt/src/mqtt.c

```diff
@@ -1430,6 +1430,7 @@ MQTTStatus_t MQTT_Connect( MQTTContext_t * pContext,
     {
         LogInfo( ( "MQTT connection established with the broker." ) );
         pContext->connectStatus = MQTTConnected;
+        pContext->keepAliveIntervalSec = pConnectInfo->keepAliveSeconds;
```

Comment:
Keep alive was not working before? How was this caught?

Reply:
Integration tests: https://github.com/aws/aws-iot-device-sdk-embedded-C/pull/1013

Reply:
ok. Thanks. Good that we added that check to unit tests as well.